TimedArithmeticQuiz/src/components/Question.jsx

```diff
@@ -0,0 +1,17 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+export const Question = React.memo(function Question({ questionExpression, enteredAnswer, setEnteredAnswer, onNextClick }) {
+  return <div className="question margin-top-12">
+    <div className="questionExpression">{questionExpression} =</div>&nbsp;
+    <input type="number" value={enteredAnswer? enteredAnswer : ""} onChange={e => setEnteredAnswer(e.target.value)} />&nbsp;
```

Comment:
Why do we need a check for enteredAnswer here? Can the default value not suffice?

Reply:
yes, will update

Reply:
done